packages/firestore/test/unit/util/async_queue.test.ts

```diff
@@ -143,7 +143,8 @@ describe('AsyncQueue', () => {
     const doStep = (n: number): Promise<number> =>
       defer(() => completedSteps.push(n));
     queue.enqueueAndForget(() => doStep(1));
-    const last = queue.enqueueAfterDelay(timerId1, 5, () => doStep(4));
+    // Flaky on Safari, increasing delay to 1000ms to try to increase reliability.
+    const last = queue.enqueueAfterDelay(timerId1, 1000, () => doStep(4));
```

Comment:
This is too long - we cannot wait a full second for a single test. Can we skip this test in Safari?

Reply:
Sure, adding a check.